girder/molecules/server/molecule.py

```diff
@@ -395,7 +395,9 @@ def search(self, params):
             else:
                 r.raise_for_status()
 
-            (inchi, inchikey) = openbabel.to_inchi(r.content.decode('utf8'), 'sdf')
+            sdf_data = r.content.decode('utf8')
+
+            (inchi, inchikey) = openbabel.to_inchi(sdf_data, 'sdf')
```

Comment:
We might even be able to get rid of some of these next eight lines or so as well...

In `create_molecule()` that is called below, it converts it to inchi and inchikey, and checks to see if the molecule exists. So I don't know if we need to do that here.

Reply:
Thanks for the hint I cleaned it up further